cfgov/scripts/initial_test_data.py

```diff
@@ -95,3 +96,11 @@ def publish_page(child=None, root=site_root):
     if not LearnPage.objects.filter(title='Learn Page'):
         lp = LearnPage(title='Learn Page', slug='learn-page', owner=admin_user)
         publish_page(lp, bfp)
+
+    # Molecules
+    bfp.header.stream_data = [{'type': 'text_introduction', 'value': {'body': '', 'has_rule': False, 'intro': 'this is an intro', 'heading': '', 'links': []}}]
```

Comment:
Not really, because that only creates it. If it already exists it's not retrieved.